lib/Fmt.mli

```diff
@@ -54,6 +54,9 @@ val protect : t -> on_error:(exn -> unit) -> t
 val break : int -> int -> t
 (** Format a break hint. *)
 
+val force_break : t
+(** [force_break] forces a break with indentation 0. *)
```

Comment:
Mention that is is equivalent to `break 1000 0` ? Written like this, it could be confused with `@\n`.